src/com/android/launcher3/widget/BaseWidgetSheet.java

```diff
@@ -163,12 +163,12 @@ public final void onClick(View v) {
             if (mWidgetCellWithAddButton != wc) {
                 // If click is on a cell not showing an add button, show it now.
                 final PendingAddItemInfo info = (PendingAddItemInfo) wc.getTag();
-                if (mActivityContext instanceof Launcher) {
-                    wc.showAddButton((view) -> addWidget(info));
-                } else {
-                    wc.showAddButton((view) -> mActivityContext.getItemOnClickListener()
-                            .onClick(wc));
-                }
+//                if (mActivityContext instanceof Launcher) {
```

Comment:
why are we commenting this? is there no work arround?

Reply:
this is the code that makes that artifact appear when clicking on widgets without pressing them, I actually don't know why this is useful, so I just commented it so I don't lose sight of it

Reply:
> this is the code that makes that artifact appear when clicking on widgets without pressing them, I actually don't know why this is useful, so I just commented it so I don't lose sight of it
> 
> The behavior is shown in the video

It's likely part of Android 15 feature here

![image](https://github.com/user-attachments/assets/0f43a7a3-3ce1-488d-b3f9-2b24f2a4fc12)

Reply:
Thanks for the info, I'll update the PR taking this into consideration

Reply:
Do you know if this is only available in Android 15, or is it also present in Android 14 or other versions?

Reply:
@MrSluffy should i isolate this by Android Version ?? as i thinkt this is not expected to work before a15?

Reply:
> Do you know if this is only available in Android 15, or is it also present in Android 14 or other versions?

Hello, sorry for the slow response. This feature or "hint" is only available in Android 15 (as of Beta 2)